lib/Zonemaster/Backend/TestAgent.pm

```diff
@@ -132,7 +138,23 @@ sub run {
         }
     }
 
-    $self->{_db}->store_results( $test_id, Zonemaster::Engine->logger->json( 'INFO' ) );
+    #Zonemaster::Backend::Metrics::timing("zonemaster.testagent.log_callback_duration",  $callback_duration * 1000);
+
+    my $start_time_2 = [ gettimeofday ];
+
+    # TODO: Make minimum level configurable
+    my @entries = grep { $_->numeric_level >= $numeric{INFO} } @{ Zonemaster::Engine->logger->entries };
+
+    Zonemaster::Backend::Metrics::timing("zonemaster.testagent.log_callback_add_result_entry_grep_duration",  tv_interval($start_time_2) * 1000);
```

Comment:
Maybe rename to `log_callback_add_result_entry_filter_duration`

Reply:
Suggestion applied.